docs/config.rst

```diff
@@ -384,7 +384,7 @@ Configuration Options
   * Importance: low
 
 ``schema.registry.zk.namespace``
-  The string that is used as the zookeeper namespace for storing Schema Registry metadata. SchemaRegistry instances which are part of the same Schema Registry service should have the same ZooKeeper namespace.
+  The string that is used as the |zk| namespace for storing schema registry metadata. Schema Registry instances which are part of the same schema registry service should have the same |zk| namespace.
```

Comment:
don't we treat schema registry as a proper noun such that it should always be capitalized?

Reply:
yes, i will replace with the `|sr|`string.